fastasyncworldedit/configuration/permissions.md

```diff
@@ -0,0 +1,47 @@
+# Permissions
+
+Table of Contents
+
+[🏠 Return to the index](../README.md)
+
+- [Admin permissions](#admin-permissions)
+- [User permissions](#user-permissions)
+- [Region permissions](#region-permissions)
+- [Extended WorldEdit permissions](#extended-worldedit-permissions)
+
+> [TIP] **Info:** For command related permissions, see [here](../basic-commands/main-commands-and-permissions.adoc).
+
+## Admin permissions
+
+- `fawe.admin` (allow bypassing through `/wea` command)
+- `fawe.bypass` (automatically bypass worldedit restrictions)
+- `worldedit.anyblock` (bypass worldedit's `disallowed-blocks`)
+- `worldedit.inventory.unrestricted` (bypass inventory restrictions if those are enabled)
+
+## User permissions
+
+- `fawe.limit.<limitgroup>` (Grants the user the specific group of limits.)
+- [`fawe.permpack.basic`](https://github.com/IntellectualSites/FastAsyncWorldEdit/blob/main/worldedit-bukkit/src/main/resources/plugin.yml#L31) (A bunch of worldedit perms which are okay to use on a creative server)
```

Comment:
This should be a relative URL to the corresponding gitbook page.

Reply:
I can't seem to find the required file under the same repo, is it needed to point to some other page/file?

Reply:
Wrong line I commented at, please ignore it :)